v0/destinations/canny/util.js

```diff
@@ -0,0 +1,81 @@
+const qs = require("qs");
+const { httpPOST } = require("../../../adapters/network");
+const { CustomError } = require("../../util");
+
+/**
+ * Function to retrieve userId from canny using axios
+ * @param apiKey
+ * @param message
+ * @returns canny userId
+ */
+const retrieveUserId = async (apiKey, message) => {
+  try {
+    if (message?.context?.externalId?.type === "cannyUserId") {
```

Comment:
Use this function instead `getDestinationExternalID`.

Reply:
Done